.github/workflows/pr.yaml

```diff
@@ -54,11 +54,65 @@ jobs:
+            echo "::endgroup::" && \
+            exit 1 )
+
+      - name: Save bundle diff as patch
+        id: bundle-diff-patch-artifact-upload
+        uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() && steps.bundle-diff-checker.outcome == 'failure' }}
+        with:
+          name: bundle-diff-patch
+          path: bundle.pr-${{ github.event.number }}.patch
+          retention-days: 5
+
+      - name: Comment on PR if bundle manifests are outdated
+        uses: actions/github-script@v7
+        if: ${{ !cancelled() && steps.bundle-diff-checker.outcome == 'failure' }}
+        # TODO(rm3l): this won't work for fork PRs due to permission restrictions. Remove this once this is fixed for fork PRs.
+        continue-on-error: true
+        with:
+          script: |
+            await github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '⚠️ <b>Files changed in bundle generation<b>!<br/><br/>Please make sure to regenerate the bundle with `make bundle` and push the changes. Make sure you unset any related env vars like `VERSION` or `IMAGE_TAG_BASE` or `IMG` before running this command, as they may affect the resulting manifests.<br/>For your convenience, the diff is attached as a job artifact [here](${{ steps.bundle-diff-patch-artifact-upload.outputs.artifact-url }}), so you can easily download and Git-apply it right away instead of running `make bundle`.<br/>You might also need to manually update the CSV in [`.rhdh/bundle/manifests/rhdh-operator.csv.yaml`](.rhdh/bundle/manifests/rhdh-operator.csv.yaml) file accordingly.'
+            })
+
+      - name: Comment on PR if bundle manifests are up-to-date
+        uses: actions/github-script@v7
+        if: ${{ !cancelled() && steps.bundle-diff-checker.outcome == 'success' }}
+        # TODO(rm3l): this won't work for fork PRs due to permission restrictions. Remove this once this is fixed for fork PRs.
+        continue-on-error: true
+        with:
+          script: |
+            await github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '🎉 <b>Bundle manifests are up-to-date!<br/><br/>Please also review the bundle manifests to make sure manual updates to the CSV in [`.rhdh/bundle/manifests/rhdh-operator.csv.yaml`](.rhdh/bundle/manifests/rhdh-operator.csv.yaml) file are not required.'
+            })
+
       # gosec needs a "build" stage so connect it to the lint step which we always do
-      - name: build
```

Comment:
see comment above -- `gosec needs a "build" stage so connect it to the lint step which we always do`